### DISCUSSION TITLE
A "tools page".. with currency converter

### DISCUSSION THREAD

**Discussion by softsimon** [mempool/mempool#554] [Ideas]
What I want is mainly a currency converter where I can convert between BTC and other fiat currencies, but also between satoshis and BTC and see how much $$$ some satoshis amount is. 

This must be run locally after downloading the latest currency rates to protect the user privacy.

Currently I have to google to find a tool like this, or use the coinmarketcap converter that now sends all requests to the server.

**Comment by wiz** [comment]
I gotta admit, I currently just google search "0.01 BTC to USD" or "500 USD to BTC" which is quite handy, but it would be nice to have a tools page on mempool for various advanced stuff
